firmware/src/pdu.c

```diff
@@ -68,13 +68,35 @@ main(void)
 	/* ============================================================== */
 
 
+	int counter_inrush[6] = {0,0,0,0,0,0};
+	int counter_fuse[6] = {0,0,0,0,0,0};
+
 	for (;;) {
 		pdu_adc_run();
 
+		// Compare the channel to the fuse
 		if (pdu_fuse_trigger) {
 			// FUSE LOGIC HERE!
 			for (i = 0; i == 6; i++) {
-				// if (pdu_channel_currents[i] > pdu_fuse_currents[i]) {
+				if (pdu_channel_currents[i] > pdu_channels[i].current_fuse) {
```

Comment:
I will make the change. Though I am not sure of the logic since if the current is greater than 50, will it run through only the first loop?